doc/RecordingControl.xml

```diff
@@ -1684,8 +1755,193 @@ SetRecordingJobMode(JobToken, Active)
       </varlistentry>
     </variablelist>
   </appendix>
+  <appendix xml:id="_refCloudRecording">
+    <title>Object storage recording format (Normative)</title>
+    <section>
+      <title>Overview</title>
+      <para>
+        This annex describes the requirements for recording from a device to an object storage for later use by a receiver.
+        For this, the device uploads objects to the object storage.
+        The device could be a camera, the receiver could be a video player.
+        The object storage typically resides in the cloud or network.
```

Comment:
```suggestion
        The object storage typically resides in the cloud.
```
Since the sentence is already about typical usage, the most predominant is the cloud. I don't think it is necessary to specify the network variety.

Reply:
Think about this, I don't think this sentence adds value, so I just removed it completely.